[2023-07-29T22:40:33.182+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trainer.encrypt_data_worker3 manual__2023-07-29T22:26:04.728771+00:00 [queued]>
[2023-07-29T22:40:33.188+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trainer.encrypt_data_worker3 manual__2023-07-29T22:26:04.728771+00:00 [queued]>
[2023-07-29T22:40:33.189+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-07-29T22:40:33.198+0000] {taskinstance.py:1327} INFO - Executing <Task(BranchPythonOperator): encrypt_data_worker3> on 2023-07-29 22:26:04.728771+00:00
[2023-07-29T22:40:33.212+0000] {standard_task_runner.py:57} INFO - Started process 546338 to run task
[2023-07-29T22:40:33.214+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'trainer', 'encrypt_data_worker3', 'manual__2023-07-29T22:26:04.728771+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/train_dag.py', '--cfg-path', '/tmp/tmp1j6mfgab']
[2023-07-29T22:40:33.214+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask encrypt_data_worker3
[2023-07-29T22:40:33.265+0000] {task_command.py:410} INFO - Running <TaskInstance: trainer.encrypt_data_worker3 manual__2023-07-29T22:26:04.728771+00:00 [running]> on host ip-172-31-37-30.ap-south-1.compute.internal
[2023-07-29T22:40:33.371+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='trainer' AIRFLOW_CTX_TASK_ID='encrypt_data_worker3' AIRFLOW_CTX_EXECUTION_DATE='2023-07-29T22:26:04.728771+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-29T22:26:04.728771+00:00'
[2023-07-29T22:40:33.379+0000] {logging_mixin.py:150} WARNING - /home/ubuntu/theft-detection/env1/lib/python3.10/site-packages/torchvision/models/_utils.py:208 UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
[2023-07-29T22:40:33.379+0000] {logging_mixin.py:150} WARNING - /home/ubuntu/theft-detection/env1/lib/python3.10/site-packages/torchvision/models/_utils.py:223 UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=None`.
[2023-07-29T22:40:33.903+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ubuntu/theft-detection/env1/lib/python3.10/site-packages/airflow/operators/python.py", line 215, in execute
    branch = super().execute(context)
  File "/home/ubuntu/theft-detection/env1/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/ubuntu/theft-detection/env1/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/ubuntu/GenAI-Rush/GenAI-Rush-FedGen/airflow/dags/train_dag.py", line 32, in encrypt_module
    max_physical_batch_size=run_config['max_physical_batch_size'], worker=run_config['client'])
KeyError: 'max_physical_batch_size'
[2023-07-29T22:40:33.915+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=trainer, task_id=encrypt_data_worker3, execution_date=20230729T222604, start_date=20230729T224033, end_date=20230729T224033
[2023-07-29T22:40:33.925+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 12 for task encrypt_data_worker3 ('max_physical_batch_size'; 546338)
[2023-07-29T22:40:33.987+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-29T22:40:34.019+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
